msgraph-teams-demo/src/msgraph/teams/LocalTestUserWithAADid.java

```diff
@@ -7,7 +7,9 @@ public class LocalTestUserWithAADid {
 
   public static void assign(String username, String azureID) {
```

Comment:
I guess this class is just for testing/demo purpose. This should not implemented by anybody in productive code?

Reply:
yes, only for the demo ... otherwise we can't play with azure in the designer